src/Chapter3.hs

```diff
@@ -1081,7 +1081,9 @@ isWeekend Sunday   = True
 isWeekend _        = False
 
 nextDay :: Weekday -> Weekday
-nextDay = succ
+nextDay Sunday    = Monday
```

Comment:
That is right!
You could also use the instance of `Bounded` in here:

```suggestion
    | day == maxBound = minBound
```

This is also a bit more maintainable. If you decide, for example, to have the "American" system of week days in the data type constructors, you won't need to change this function at all! 🙂